### PR TITLE
Update liquidity rewards display

### PR DESCRIPTION
**Pull request by jonahlipsky**
Change Summary:
* refactor rewards per block into its own class
* add a mention of exactly how much MS per block you receive
    ** NOTE: THE AMOUNTS DON'T LOOK QUITE RIGHT. Close, but not exact.

Screen Shot 2021-04-26 at 3 14 36 PM

### CODE REVIEW

File: src/LiquidityBalances.js [hunk 1]
```diff
@@ -90,21 +91,13 @@ export default function LiquidityBalances(props) {
                       <td className="text-right" rowSpan="2">
                         <Percentage
                           numerator={liquidityToken.balance}
-                          denomiator={liquidityToken.totalSupply}
+                          denominator={liquidityToken.totalSupply}
+                        />
+                        <RewardPerBlock
+                          liquidityToken={liquidityToken}
+                          blockNumber={blockNumber}
+                          MS={MS}
```
Comment (masonforest) [inline src/LiquidityBalances.js:99]: What do you think about importing MS directly in `RewardPerBlock` since it's a constant?

**Review by masonforest — APPROVED** [review]
This looks good man! 🎉 

I like the extraction of the component! I've been gong back and forth about using functions like the `value` function or components for text-only stuff like this. As this matures it'd probably be good to standardize around components like you've done here...

Now we just need to figure out what that header title should be. Agreed that what we have now is a bit wordy but probably fine for now...

Nice work! 🍇

File: src/RewardPerBlock.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  ).toFixed(6);
+
+  return (
+    <>
+      {liquidityToken.totalSupply &&
+      blockReward(blockNumber, liquidityToken.tokenAddress)
+        ? ` (${value(issuance, MS.address, {
+            showCurrency: true,
+            zeroString: "N/A",
+          })}) = ${yourIssuance} MS`
```
[the remark this answers is not in the document]
Comment (jonahlipsky) [reply src/RewardPerBlock.js:26]: But here I'm adding another piece of information. In this refactor, the first number is the amount of token issued to the pool per block, which I think is nice information! The second number is how much you get.

**Review by masonforest — APPROVED** [review]
